client/jetpack-connect/style.scss

```diff
@@ -628,6 +628,11 @@
 	&[type='password'] {
 		margin-bottom: 0;
 	}
+
+	&.is-primary {
+		background-color: #00be28;
+		border-color: #007218;
```

Comment:
Let's use the variables like here:

https://github.com/Automattic/wp-calypso/blob/d61afbb8e6393a9e6d161893be3abfa14adaa008/client/jetpack-connect/style.scss#L27-L28

Reply:
Done 👍 Thanks.